blog.qmd

```diff
@@ -0,0 +1,65 @@
+---
```

Comment:
@Robinlovelace I assume this is for the geocompx blog?

Reply:
Good plan.